core/src/main/java/com/cloud/resource/ServerResourceBase.java

```diff
@@ -306,4 +309,13 @@ public boolean start() {
     public boolean stop() {
         return true;
     }
+
+    public String calculateCurrentChecksum(String name, String path) {
+        String cloudScriptsPath = Script.findScript("", path);
```

Comment:
Is there a name of script/file that should be passed here?

Reply:
checksum of the cloud-scripts.tgz file is calculated using this method